SU-SWS/stanford_profile
# D8CORE-1351: Add Paranoia

# READY FOR REVIEW

# Summary
- See https://github.com/SU-SWS/acsf-cardinalsites/pull/98 - needed for adding Paranoia so that config sync will work.

# Review By (Date)
- Friday, March 13

# Urgency
- Needed for Paranoia. End of sprint would be nice.

# Steps to Test

1. Check out `D8CORE-1351` of the cardinalsites stack.
2. `composer install` (or `composer nuke`)
3. `blt di`
4. Ensure that Drupal installs correctly

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- JIRA ticket: D8CORE-1351
- Other PRs: https://github.com/SU-SWS/acsf-cardinalsites/pull/98

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
